winlogtimeline/ui/ui.py

```diff
@@ -456,9 +457,22 @@ def __init__(self, parent, **kwargs):
         self.tool_menu = Menu(self, **kwargs)
         self.add_cascade(label='Tools', menu=self.tool_menu, underline=0)
         # Tools -> Timeline Colors
-        self.tool_menu.add_command(label='Timeline Colors', command=self.color_settings_function, underline=0)
+        self.tool_menu.add_command(label='Timeline Colors', command=self.color_settings_function, underline=0,
+                                   accelerator='Ctrl+C')
         # Tools -> Import Log
-        self.tool_menu.add_command(label='Import Log File', command=self.import_button_function, underline=0)
+        self.tool_menu.add_command(label='Import Log File', command=self.import_button_function, underline=0,
+                                   accelerator='Ctrl+I')
+        parent.bind('<Control-i>', self.import_button_function)
```

Comment:
Having keyboard shortcuts seems a bit excessive for this functionality, in my opinion.